scripts/exregional_process_radarref.sh

```diff
@@ -235,13 +238,12 @@ cp_vrfy $BUFR_TABLE prepobs_prep.bufrtable
 #-----------------------------------------------------------------------
 
 cat << EOF > mosaic.namelist
- &setup
-  tversion=1,
-  bkversion=1,
-  analysis_time = ${YYYYMMDDHH},
-  dataPath = './',
- /
-
+   &setup
+    tversion=1,
+    bkversion=1,
+    analysis_time = ${YYYYMMDDHH},
+    dataPath = './',
+   /
```

Comment:
This "cat <<EOF" heredoc can be exempt from indent because for some applications unexpected leading whitespaces may cause problems. I will leave it to you to decide whether to change them back or not.

Reply:
OK